chore(docs): u-files - minor punctuation and grammar fixes (#18762)
gatsbyjs/gatsby: docs/docs/using-graphql-playground.md

## Intro

> On this page, we want to introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).

Reviewer: ```suggestion
This page will introduce you to an alternative to the current IDE for your GraphQL queries: [GraphQL Playground](https://github.com/prisma/graphql-playground).
```
Might as well update this for the Style Guide.

Author: Thank you!
